src/assets/styles/base.scss

```diff
@@ -315,6 +315,20 @@ ul.list {
   border-left: 1px solid $gray6;
 }
 
+.message-box {
```

Comment:
Sev, you're thinking that we can probably re-use this class for different v-cards in the future? That's why you added it here?

Reply:
I copied this class from Create UI. It's also in Edit UI. So it is part of our global / common / "base" styling.

Yes, it's for any other yellow message boxes we might have.

Reply:
I see. OK, thanks. Resolving.